content/en/blog/_posts/2020-08-14-introducing-hierarchical-namespaces.md

```diff
@@ -161,24 +161,22 @@ _also_ allow ingress traffic between all of those namespaces. The “tree” lab
 can only be applied by HNC, and is guaranteed to reflect the latest hierarchy.
 
 You can learn all about the features of HNC from the [user
-guide](https://github.com/kubernetes-sigs/multi-tenancy/tree/master/incubator/hnc/docs/user-guide).
+guide](https://github.com/kubernetes-sigs/hierarchical-namespaces/tree/master/docs/user-guide).
 
 # Next steps and getting involved
 
 If you think that hierarchical namespaces can work for your organization, [HNC
-v0.5.1 is available on
-GitHub](https://github.com/kubernetes-sigs/multi-tenancy/releases/tag/hnc-v0.5.1).
+v1.0.0 is available on
+GitHub](https://github.com/kubernetes-sigs/hierarchical-namespaces/releases/tag/v1.0.0).
```

Comment:
Don't retrospectively change the tag. Can we link to the 0.5.1 release still?

Reply:
Understood. The change was made in order to update the links as it might be somewhat misleading for someone getting introduced to HNC.